recipes-omnect/images/omnect-os-image.bb

```diff
@@ -45,25 +45,36 @@ IMAGE_BASENAME = "omnect-os"
 
 IMAGE_NAME = "${DISTRO_NAME}_${DISTRO_VERSION}_${MACHINE}"
 
+EXTRA_PACKAGES_CELLULAR = "\
+    libqmi \
+    libmbim \
+    mobile-broadband-provider-info \
+    networkmanager networkmanager-wwan \
+    modemmanager \
+    usb-modeswitch usb-modeswitch-data \
+"
+
 IMAGE_INSTALL = "\
+    ${@bb.utils.contains('MACHINE_FEATURES', '3g', '${EXTRA_PACKAGES_CELLULAR}', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'iotedge', ' aziot-edged iotedge kernel-modules', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', ' systemd-bash-completion', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'wifi-commissioning', ' wifi-commissioning-gatt-service', '', d)} \
     ${CORE_IMAGE_BASE_INSTALL} \
     bootloader-env \
     coreutils \
+    e2fsprogs-tune2fs \
     iot-hub-device-update \
     iptables \
+    jq \
     kernel-image \
     kmod \
+    less \
```

Comment:
we had that discussion already, I added it for convenience purposes because the minimal pager implemented in busybox is really awkward when it comes to searching and that is really annoying when investigating on device logs on the device.

Reply:
actually, wanting to have less is the same reason as using bash instead of sh, it makes life easier.
if you insist on having it only for development, so be it ...